content/pda.md

```diff
@@ -0,0 +1,523 @@
+# PDAs
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
+
+- Explain Program Derived Addresses (PDAs)
+- Explain various use cases of PDAs
+- Describe how PDAs are derived
+- Use PDA derivations to locate and retrieve data
+
+# TL;DR
+
+- A Program Derived Address (PDA) is derived from a program ID and an optional list of seeds.
+- PDAs are designed to be controlled by the program they are derived from
+- PDAs provide a deterministic way to find data stored on accounts
+- Programs can sign instruction for PDAs derived from its program ID
```

Comment:
@ZYJLiu plural instructions?  Also, rather than "...for PDAs derived from..." how about

> "...on behalf of PDAs derived from..."